tests/system/tests/transfers/watch.only.transfer.system.test.ts

```diff
@@ -0,0 +1,17 @@
+import * as allure from 'allure-js-commons';
+import { test } from '../../utils/baseRegularFixture';
+
+
+const feature = 'Wallets. Watch-only. Single wallet';
+const story = 'Transfers';
+
+test.describe('Watch-only transfers', { tag: ['@watch-only-transfers', '@regress'] }, () => {
+  test('Watch-only transfer', async ({ loginPage }) => {
+    await allure.feature(feature);
+    await allure.story(story);
+    const watchOnlyWallet = await loginPage.createBaseWatchOnlyWallet();
+    const assetsPage = await watchOnlyWallet.gotoMain();
+    await assetsPage.checkTransferButtonNotExists();
```

Comment:
here also need to check that the direct link won't open a transfer widget, check "Polkadot Vault, single wallet, can make regular transfer on ${chainName}"